cli/mender-artifact/partition.go

```diff
@@ -324,7 +357,7 @@ func newFatFile(imageFilePath string, partition partition) (*fatFile, error) {
 
 // Read Dump the file contents to stdout, and capture, using MTools' mtype
 func (f *fatFile) Read(b []byte) (n int, err error) {
-	cmd := exec.Command("mtype", "-i", f.path, "::/"+f.imageFilePath)
```

Comment:
Why did we need to change this?, isn't `imageFilePath` same as before? I see that in `Delete` we do trim the last `/`, but not in REad or Write. Is that intentional?